Date.cpp

```diff
@@ -11,7 +11,7 @@ date::date(std::string init, const bool AUC)
 		return;
 	}
 
-	if (init[0] == '\"')
+	if (init.starts_with('\"'))
```

Comment:
Is there a reason not to just use remQuotes()?

Reply:
I don't see one, replaced.